docker/config/services/dev.musicbrainz-server.json

```diff
@@ -1,7 +1,7 @@
 {
   "DBDefs": {
     "CATALYST_DEBUG": "0",
-    "DB_SCHEMA_SEQUENCE": "29",
+    "DB_SCHEMA_SEQUENCE": "30",
```

Comment:
Side note: I'm pretty sure this file is unused, but we can look into that later.